packages/embed/workers-site/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "private": true,
+  "version": "1.0.0",
```

Comment:
Ah cool, no "name" field in here. That makes sense, I had to update the other workers sites names bc metro was complaining

Reply:
TIL